mongo_orchestration/server.py

```diff
@@ -134,7 +163,13 @@ def main():
     if args.command == 'stop':
         daemon.stop()
     if args.command == 'start' and not args.no_fork:
-        daemon.start()
+        pid = daemon.start()
+        if not await_connection(host=args.bind, port=args.port):
+            print(
+                'Could not connect to mongo-orchestration on %s:%d (pid: %d) '
```

Comment:
I think this message might be confusing. Users may think "why if mongo-orchestration trying to connect to itself?"